pinot-controller/src/test/java/com/linkedin/pinot/controller/api/restlet/resources/TableViewsTest.java

```diff
@@ -0,0 +1,137 @@
+import static org.testng.Assert.assertNull;
+
+
+public class TableViewsTest extends ControllerTest {
+
+  public static final String TABLE_NAME = "VIEWS_TABLE";
+
+  @BeforeClass
+  public void setupTest()
+      throws Exception {
+    startZk();
+    startController();
+    ControllerRequestBuilderUtil.addFakeBrokerInstancesToAutoJoinHelixCluster(getHelixClusterName(),
+        ZkStarter.DEFAULT_ZK_STR, 5, true);
+    ControllerRequestBuilderUtil.addFakeDataInstancesToAutoJoinHelixCluster(getHelixClusterName(),
+        ZkStarter.DEFAULT_ZK_STR, 20, true);
+
+    JSONObject request = ControllerRequestBuilderUtil.buildBrokerTenantCreateRequestJSON("default", 5);
+    sendPostRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forBrokerTenantCreate(),
+        request.toString());
+    request = ControllerRequestBuilderUtil.buildServerTenantCreateRequestJSON("default", 20, 16, 2);
+    sendPostRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forBrokerTenantCreate(),
+        request.toString());
+
+    request = ControllerRequestBuilder.buildCreateOfflineTableJSON(TABLE_NAME, "default", "default",
+        2, "BalanceNumSegmentAssignmentStrategy");
+    sendPostRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forTableCreate(),
+        request.toString());
+    JSONObject metadata = new JSONObject();
+    metadata.put("streamType", "kafka");
+    metadata.put(DataSource.STREAM_PREFIX + "." + DataSource.Realtime.Kafka.CONSUMER_TYPE, DataSource.Realtime.Kafka.ConsumerType.highLevel.toString());
+    metadata.put(DataSource.STREAM_PREFIX + "." + DataSource.Realtime.Kafka.TOPIC_NAME, "fakeTopic");
+    metadata.put(DataSource.STREAM_PREFIX + "." + DataSource.Realtime.Kafka.DECODER_CLASS, "fakeClass");
+    metadata.put(DataSource.STREAM_PREFIX + "." + DataSource.Realtime.Kafka.ZK_BROKER_URL, "fakeUrl");
+    metadata.put(DataSource.STREAM_PREFIX + "." + DataSource.Realtime.Kafka.HighLevelConsumer.ZK_CONNECTION_STRING, "potato");
+    metadata.put(DataSource.Realtime.REALTIME_SEGMENT_FLUSH_SIZE, Integer.toString(1234));
+    metadata.put(DataSource.STREAM_PREFIX + "." + DataSource.Realtime.Kafka.KAFKA_CONSUMER_PROPS_PREFIX + "." + "auto.offset.reset",
+        "smallest");
+    request = ControllerRequestBuilder.buildCreateRealtimeTableJSON(TABLE_NAME, "default", "default",
+        "potato", "DAYS", "DAYS", "5", 2, "BalanceNumSegmentAssignmentStrategy", metadata, "fakeSchema", "fakeColumn",
+        Collections.<String>emptyList(), "MMAP", true);
+    sendPostRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forTableCreate(),
+        request.toString());
+  }
+
+  @AfterClass
+  public void teardownTest()
+      throws Exception {
+    stopController();
+    stopZk();
+  }
+
+  private void addOneSegment(String resourceName) {
+    final SegmentMetadata segmentMetadata = new SimpleSegmentMetadata(resourceName);
+    //addSegment(segmentMetadata, "downloadUrl");
```

Comment:
No op?  Need to beef up this test with segment so that we can check for contents of IS and EV.